assignment/src/com/assignmentweek2/person/Student.java

```diff
@@ -0,0 +1,41 @@
+package com.assignmentweek2.person;
+
+public class Student  extends Person{
+    protected String name;
+    protected String address;
+    public Student(String name, String address) {
+        this.name = name;
+        this.address = address;
+    }
+    public String getName() {
+        return name;
+    }
+    public void setName(String name) {
+        this.name = name;
+    }
+    public String getAddress() {
+        return address;
+    }
+    public void setAddress(String address) {
+        this.address = address;
+    }
+     void Studentdisplay(){
+        System.out.println("Student Role: "+getName());
+    }
+    @Override
+    void Teacherdisplay() {
+    }
+    @Override
+    void Doctordisplay() {
+    }
```

Comment:
Delete these methods after you change Person method names common for all types.

Reply:
deleted